doc/lightning-fetchinvoice.7.md

```diff
@@ -0,0 +1,90 @@
+lightning-fetchinvoice -- Command for fetch an invoice for an offer
+===================================================================
+
+SYNOPSIS
+--------
+
+*EXPERIMENTAL_FEATURES only*
+
+**fetchinvoice** *offer* \[*msatoshi*\] \[*quantity*\] \[*recurrence_counter*\] \[*recurrence_start*\] \[*recurrence_label*\] \[*timeout*\]
+
+DESCRIPTION
+-----------
+
+The **fetchinvoice** RPC command contacts the issuer of an *offer* to get
+an actual invoice that can be paid.  It highlights any changes between the
+offer and the returned invoice.
+
+The offer must not contain *send_invoice*; see lightning-sendinvoice(7).
+
+*msatoshi* is optional: it is required if the *offer* does not specify
```

Comment:
`is optional: it is required` is contradictory -> `*msatoshi* is required if the *offer* does not specify, otherwise it MUST be omitted`

Reply:
Same for the fields below.